tests/testthat/test-get_variance.R

```diff
@@ -284,3 +282,25 @@ test_that("random effects CIs, poly slope", {
     ignore_attr = TRUE
   )
 })
+
+
+
+test_that("fixed effects variance for rank-deficient models, #765", {
+  skip_if_not_installed("glmmTMB", minimum_version = "1.1.7.9000")
+  set.seed(101)
```

Comment:
@bbolker I added a test that relies on the dev-version of glmmTMB - is this safe, or should I set the minimum version to 1.1.8? This is related to the rank-deficiency issue (https://github.com/glmmTMB/glmmTMB/issues/522).